docs/advanced-usage.md

```diff
@@ -131,3 +132,57 @@ staticAnalysis {
 Please note that this is not yet supported for Detekt.
 
 [penaltyextensioncode]: https://github.com/novoda/gradle-static-analysis-plugin/blob/master/plugin/src/main/groovy/com/novoda/staticanalysis/PenaltyExtension.groovy
+
+
+### Consume rules from artefact     
+
+In order to reuse your rules among multiple projects or to easily use an open source rule set, we added support for consuming the 
+rules for all supported tools from a maven artefact. An example of how to create and publish your rules as artefact can be 
+found [here](https://github.com/novoda/novoda/blob/master/scaffolding/build.gradle). Novoda open sourced their rules to [bintray](https://bintray.com/novoda/maven/static-analysis-rules) including
+configurations for Checkstyle, PMD, FindBugs, detekt and Android Lint.
+
+
+To consume the rules from an artefact you need to configure the `rules` closure pointing to a maven artefact and use it when configuring the tools, e.g.
+ 
+```gradle
+staticAnalysis {
+    rules {
+        novoda {
+            maven 'com.novoda:static-analysis-rules:0.2'
+        }
+    }
+    
+    checkstyle {
+        toolVersion '8.8'
+        config rules.novoda['checkstyle-modules.xml']
+    }
+}
+```
+
+These are the necessary configurations for all supported tools:
```

Comment:
I'm not sure I understand why `necessary` is used here. Shouldn't be something like: "If you want to use the configuration from the artifact you can configure the tools as follows" and then I would list each tool in a different paragraph, as each configuration is independent from each other.